UI/src/i18n.js

```diff
@@ -0,0 +1,46 @@
+/** @format */
+/* global __SUPPORTED_LOCALES */
+
+export const SUPPORT_LOCALES = __SUPPORTED_LOCALES;
+
+import { nextTick } from "vue";
+import { createI18n } from "vue-i18n";
+
+const i18n = createI18n({
+    globalInjection: true,
+    legacy: false,
+    fallbackWarn: false,
+    missingWarn: false, // warning off
+    locale: window.lsmbConfig.language,
+    fallbackLocale: "en",
+    messages: {}
+});
+
+function setI18nLanguage(locale) {
+    // Update document
+    document.querySelector("html").setAttribute("lang", locale);
+    document.querySelector("html").setAttribute("xml:lang", locale);
```

Comment:
Since we stopped declaring our documents as `xhtml`, I think the `xml:lang` attribute is not appropriate/not required anymore?

Reply:
Have we? https://github.com/ledgersmb/LedgerSMB/blob/master/UI/lib/ui-header.html#L15 still uses xhtml

Reply:
Hmm. well, one or the other is incorrect, because `<!DOCTYPE html>` indicates HTML5 which definitely isn't xhtml...

Reply:
Thinking about it, we probably need to remove the XML namespace declaration for XHTML.

Reply:
Agreed. Doing so